lib/v2/huoxian/radar.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'huoxian.cn': {
+        _name: '火线',
+        zone: [
+            {
+                title: '火线',
```

Comment:
```suggestion
                title: 'Zone',
```
https://github.com/DIYgod/RSSHub/blob/31577b707b91f79234ab7c50b3c0855f872ea7b4/docs/bbs.md?plain=1#L678

`title` should be the same as heading level 3.
docs: https://docs.rsshub.app/joinus/new-radar.html